scripts/test-enemy.gd

```diff
@@ -1,6 +1,6 @@
 extends CharacterBody3D
 
-class_name enemy
+#class_name enemy
```

Comment:
Isn't this supposed to be big E?

Reply:
It's test-enemy.gd, will not use it anymore, ignore it.
But you're right!
Thanks for checking till the detail.

Reply:
Forgot to add information. It shows error about two Global Class exist together. Imma erase it or change it to enemy_test.